serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1 +1,209 @@
-= Building Workflow Images locally with Quarkus CLI
\ No newline at end of file
+Building Workflow Images locally with Quarkus CLI
+=================================================
+v1.0, 06-03-2022
+:compat-mode!:
+// Metadata:
+:description: Build Serverless Application with Quarkus CLI
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli
+// links
+:example-project: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-greeting-quarkus
+
+:quarkus-container-images: https://quarkus.io/guides/container-image
+:quarkus-native-builds: https://quarkus.io/guides/building-native-image
+:google-jib: https://github.com/GoogleContainerTools/jib
```

Comment:
I think we need to standardize these variables by adding a suffix to them. For example: `google-jib-url`

Reply:
sounds good.
@akumar074 please add this to the contributors' guide.